target/linux/bcm5862x/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,23 @@
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+case "$(board_name)" in
+meraki,mx64)
+	ucidef_set_led_netdev "wan" "WAN Port" "green:wan-right" "wan"
+	ucidef_set_led_netdev "lan1" "LAN Port 1" "green:lan1-right" "lan1"
+	ucidef_set_led_netdev "lan2" "LAN Port 2" "green:lan2-right" "lan2"
+	ucidef_set_led_netdev "lan3" "LAN Port 3" "green:lan3-right" "lan3"
+	ucidef_set_led_netdev "lan4" "LAN Port 4" "green:lan4-right" "lan4"
+	;;
+meraki,mx65)
+	ucidef_set_led_default "lan" "LAN Port LEDs" "green:lan-leds" "1"
```

Comment:
This should not be used for exposing switches.

Reply:
Would there be an alternative appropriate way to go about providing an on/off toggle for all switch port LEDs? I know @Ansuel is working on [qca8k LED control](https://github.com/openwrt/openwrt/pull/3954/commits/528ee5ebba7e1d0917a7080957481789bd5dbeae#diff-b5b6867a615b77111c560cce642fa72352f0da23c2c4a380dad3e31475bc6ca2) but this also applies at switch level and doesn't allow individual port LED control or toggling on/off as yet.

Reply:
The support can be expanded (actually not that hard). What I really need is an example of some dsa driver that define led control just to not reinvent the wheel and follow a standard upstream approach. From documentation each port can be controlled individually. (in my initial patch I only added support for blink freq and the same option is applied to every led)

Reply:
@adschm I believe it is also not correct to state the lan port leds in the device tree led bindings as well, since this is controlling a toggle for a group of leds and not controlling an individual led. Correct me if I am wrong. Have removed in both places.